cmd/fleetctl/apple_mdm.go

```diff
@@ -44,124 +52,136 @@ func appleMDMCommand() *cli.Command {
+			org := c.String("org")
+			apnsKeyPath := c.String("apns-key")
+			scepCACertPath := c.String("scep-cert")
+			scepCAKeyPath := c.String("scep-key")
+
+			fmt.Fprintf(
+				c.App.Writer,
+				`Sending certificate signing request (CSR) for Apple Push Notification service (APNs) to %s...
+Generating APNs key, Simple Certificate Enrollment Protocol (SCEP) certificate, and SCEP key...
+`,
+				email,
+			)
+
+			// create apns csr and send to fleetdm.com
+			apnsCSR, apnsKey, err := apple_mdm.GenerateAPNSCSRKey(email, org)
 			if err != nil {
-				return fmt.Errorf("creating SCEP CA: %w", err)
+				return fmt.Errorf("generate apns csr: %w", err)
 			}
-			const (
-				certPath = "fleet-mdm-apple-scep.crt"
-				keyPath  = "fleet-mdm-apple-scep.key"
-			)
-			if err := os.WriteFile(certPath, certPEM, 0o600); err != nil {
-				return fmt.Errorf("write %s: %w", certPath, err)
+
+			apnsKeyPEM := apple_mdm.EncodePrivateKeyPEM(apnsKey)
+
+			err = os.WriteFile(apnsKeyPath, apnsKeyPEM, 0600)
+			if err != nil {
+				return fmt.Errorf("write private key: %w", err)
 			}
-			if err := os.WriteFile(keyPath, keyPEM, 0o600); err != nil {
-				return fmt.Errorf("write %s: %w", keyPath, err)
+
+			client := fleethttp.NewClient(fleethttp.WithTimeout(10 * time.Second))
+
+			err = apple_mdm.GetSignedAPNSCSR(client, apnsCSR)
+			if err != nil {
+				return fmt.Errorf("get signed apns csr: %w", err)
 			}
-			fmt.Printf("Successfully generated SCEP CA: %s, %s.\n", certPath, keyPath)
-			fmt.Printf("Set FLEET_MDM_APPLE_SCEP_CA_CERT_PEM=$(cat %s) FLEET_MDM_APPLE_SCEP_CA_KEY_PEM=$(cat %s) when running Fleet.\n", certPath, keyPath)
-			return nil
-		},
-	}
-}
 
-func appleMDMSetupAPNSCommand() *cli.Command {
-	return &cli.Command{
-		Name:  "apns",
-		Usage: "Commands to setup APNS certificate",
-		Subcommands: []*cli.Command{
-			appleMDMSetupAPNSInitCommand(),
-			appleMDMSetupAPNSFinalizeCommand(),
-		},
-	}
-}
+			// init scep ca
+			scepCACert, scepCAKey, err := apple_mdm.NewSCEPCACertKey()
+			if err != nil {
+				return fmt.Errorf("init scep CA: %w", err)
+			}
+
+			scepCACertPEM := apple_mdm.EncodeCertPEM(scepCACert)
+			err = os.WriteFile(scepCACertPath, scepCACertPEM, 0600)
+			if err != nil {
+				return fmt.Errorf("write scep ca certificate: %w", err)
+			}
+
+			scepCAKeyPEM := apple_mdm.EncodePrivateKeyPEM(scepCAKey)
+			err = os.WriteFile(scepCAKeyPath, scepCAKeyPEM, 0600)
+			if err != nil {
+				return fmt.Errorf("write scep ca private key: %w", err)
+			}
+
+			fmt.Fprintf(
+				c.App.Writer,
+				`Success!
+
+Generated your APNs key at %s
+
+Generated your SCEP certificate at %s
+
+Generated your SCEP key at %s
+
+Go to your email to download a CSR from Fleet. Then, visit https://identity.apple.com/pushcert to upload the CSR. You should receive an APNs certificate in return from Apple.
+
+Next, use the generated certificates to deploy Fleet with `+"`mdm`"+` configuration: https://fleetdm.com/docs/deploying/configuration#mdm
```

Comment:
This may need to be updated later, but adding documentation is part of https://github.com/fleetdm/fleet/issues/8595. Perhaps I should leave a TODO to update this as part of #8595.

Reply:
Note that for now (until MDM is prod-ready), the header for the docs is:

> #### MDM (mobile device management) - IN PROGRESS

so the fragment would be `#mdm-mobile-device-management-in-progress`.